Quarkus devmode crashes on JDK >= 22 (Windows)

### Describe the bug

Simple Quarkus app crashes on Windows using JDK22:

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007ffec08ba418, pid=39660, tid=40340
#
# JRE version: Java(TM) SE Runtime Environment (22.0+36) (build 22+36-2370)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (22+36-2370, mixed mode, emulated-client, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, windows-amd64)
# Problematic frame:
# V  [jvm.dll+0x45a418]
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# C:\Users\fouad\IdeaProjects\quarkus-3.9.0.RC2-with-JDK22\hs_err_pid39660.log
[0.986s][warning][os] Loading hsdis library failed
#
# If you would like to submit a bug report, please visit:
#   https://bugreport.java.com/bugreport/crash.jsp
#

> Task :quarkusDev FAILED

FAILURE: Build failed with an exception.
```


### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

Reproducer: https://github.com/Eng-Fouad/quarkus-3.9.0.RC2-with-JDK22

### Output of `uname -a` or `ver`

Windows 11

### Output of `java -version`

22

### Quarkus version or git rev

3.9.0.CR2

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

gradle-8.7-rc-3

### Additional information

_No response_

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -202,8 +202,8 @@
         <!-- Check the compatibility matrix (https://github.com/opensearch-project/opensearch-testcontainers) before upgrading: -->
         <opensearch-testcontainers.version>2.0.0</opensearch-testcontainers.version>
         <com.dajudge.kindcontainer>1.4.5</com.dajudge.kindcontainer>
-        <aesh.version>2.7</aesh.version>
-        <aesh-readline.version>2.4</aesh-readline.version>
+        <aesh.version>2.8.2</aesh.version>
+        <aesh-readline.version>2.6</aesh-readline.version>
         <jansi.version>2.4.0</jansi.version> <!-- Keep in sync with aesh-readline and dekorate -->
         <jgit.version>6.10.0.202406032230-r</jgit.version>
         <!-- these two artifacts needs to be compatible together -->

